sonar-db/src/main/java/org/sonar/db/measure/MeasureDao.java

```diff
@@ -82,7 +83,7 @@ public MeasureDto selectByComponentKeyAndMetricKey(DbSession session, String com
   /**
    * Used by plugin Developer Cockpit
    */
-  public List<MeasureDto> selectByDeveloperForSnapshotAndMetrics(final DbSession dbSession, final long developerId, final long snapshotId, Collection<Integer> metricIds) {
+  public List<MeasureDto> selectByDeveloperForSnapshotAndMetrics(final DbSession dbSession, @Nullable final Long developerId, final long snapshotId, Collection<Integer> metricIds) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 181 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS00103)